packages/channels/src/slack/config.ts

```diff
@@ -0,0 +1,12 @@
+import Joi from 'joi'
+import { ChannelConfig } from '../base/config'
+
+export interface SlackConfig extends ChannelConfig {
+  botToken: string
+  signingSecret: string
+}
+
+export const SlackConfigSchema = {
+  botToken: Joi.string().required(),
+  signingSecret: Joi.string().required()
+}
```

Comment:
We might want to add more configs here as a future proof measure. For example ask the appId, etc. in case we need it in the future. For now these are the same configs as before